cypress/cypress/integration/topics.spec.ts

```diff
@@ -9,29 +9,82 @@ import { visitAsGuestWithDefaultSetup } from "../support/helpers";
 describe("Topics list", () => {
   it("shows topics for current mentor", () => {
     visitAsGuestWithDefaultSetup(cy, "/");
-    cy.get("[data-cy=topics]").contains("About Me");
-    cy.get("[data-cy=topics]").contains("About the Job");
+    cy.get("[data-cy=topic-tab]")
+      .trigger("mouseover")
+      .click()
+      .get("[data-cy=topics-questions-list]")
+      .within(() => {
+        cy.get("[data-cy=topic-opt-item]").contains("About Me");
+        cy.get("[data-cy=topic-opt-item]").contains("About the Job");
+      });
   });
 
   it("has default topic selected", () => {
     visitAsGuestWithDefaultSetup(cy, "/");
-    cy.get("[data-cy=topics]").find(".topic-selected").should("have.length", 1);
-    cy.get("[data-cy=topic-0]").find(".topic-selected");
+    // cy.get("[data-cy=topics]").find(".topic-selected").should("have.length", 1);
```

Comment:
ok, i see what you're asking. You really want to be testing a few things here, and none of them should involve clicking anything. You might want to break this up and test separately for mobile vs web

```typescript
it("has default topic selected (mobile)", () => {
  // test that the TOPIC tab is selected (not the history tab)
  // test that within the TOPIC dropdown that the first item is already selected
  // test that the content displays in the questions-list panel is the list of questions for the default/selected topic
})

it("has default topic selected (web)", () => {
  // test that the first topic tab is selected
  // test that the content displays in the questions-list panel is the list of questions for the default/selected topic
})
```

Reply:
https://github.com/mentorpal/mentor-client/pull/87/commits/b6e61737ebf7e2dad392557c45915a68c5bb31bf